File: Matlab/GetAtomParams.m
%Function for getting atom specfifc parameters for Rydberg code. Based on
%Matlab code written in summer of 2015 updated 24/07/2017.
%The intention is not that this be a complete database but rather that more
%parameters be added as they are needed

function [ZZ,spin,eneigval,params] = GetAtomParams(atom,nn,ll,jj) %#ok<STOUT,INUSD>
SIunits;
if strcmp(atom,'87Rb')==1
    Rb87numbers;
end
params = [alpha_c,a_1,a_2,a_3,a_4,r_c];

File: Julia/GetAtomParams.jl
#Function for getting atom specific parameters for Rydberg code. Based on Matlab code written in the summer of 2015 migrated to Julia on 23/07/2017
#The intention is not that this be a complete database but rather that more parameters be added as they are needed

function GetAtomParams(atom,nn,ll,jj)

  #Parameters for Rubidium-8
  if atom=="87Rb"
    ZZ, spin, eneigval, params, delta_nlj = Rb87numbers(nn,ll,jj)
  end

  return ZZ, spin, eneigval, params, delta_nlj
end
